hotham/src/contexts/render_context.rs

```diff
@@ -134,7 +109,11 @@ impl RenderContext {
         let pipeline_layout =
             create_pipeline_layout(vulkan_context, slice_from_ref(&descriptors.graphics_layout))?;
 
-        let shaders = Default::default();
+        let shaders = Shaders::from([
+            (VERT_NAME.into(), VERT.into()),
+            (FRAG_NAME.into(), FRAG.into()),
+            (COMPUTE_NAME.into(), COMPUTE.into()),
```

Comment:
The compute shader is created with `COMPUTE` directly in `create_compute_pipeline`. What do you think about inlining the shaders here to avoid them being used elsewhere by mistake?

Reply:
I'm not sure what you mean by this - can you elaborate?

Reply:
I was thinking something like

```rs
            (VERT_NAME.into(), include_glsl!("src/shaders/pbr.vert", target: vulkan1_1)),
            (FRAG_NAME.into(), include_glsl!("src/shaders/pbr.frag", target: vulkan1_1)),
            (COMPUTE_NAME.into(), include_glsl!("src/shaders/culling.comp", target: vulkan1_1)),
```

But it got messy with casting now that I tried it.